### PR TITLE
Handle the inst.addrepo option in the DNF payload class

### PR DESCRIPTION
**Pull request by poncovka**
Move the code from `add_additional_repositories_to_ksdata` to the DNF payload class.

### CODE REVIEW

File: pyanaconda/argument_parsing.py [hunk 1]
```diff
@@ -465,8 +465,23 @@ def __call__(self, parser, namespace, values, option_string=None):
                     help=help_parser.help_text("repo"))
     ap.add_argument("--stage2", dest="stage2", default=None, metavar="STAGE2_URL",
                     help=help_parser.help_text("stage2"))
+
+    class AppendAddRepo(Action):
```
[the remark this answers is not in the document]
Comment (VladimirSlavik) [reply pyanaconda/argument_parsing.py:469]: Should it be `ParseAddRepo` perhaps?

**Review by VladimirSlavik — APPROVED** [review]
Thank you, looks good to me.

File: pyanaconda/argument_parsing.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (poncovka) [reply pyanaconda/argument_parsing.py]: Fixed.

File: pyanaconda/argument_parsing.py [hunk 2]
```diff
@@ -465,8 +465,23 @@ def __call__(self, parser, namespace, values, option_string=None):
...
+        def __call__(self, parser, namespace, values, option_string=None):
+            try:
+                name, rest = values.split(',', maxsplit=1)
+            except ValueError:
+                raise ValueError(
+                    "The addrepo option has incorrect format. "
+                    "Use: inst.addrepo=<name>,<url>"
```
[the remark this answers is not in the document]
Comment (poncovka) [reply pyanaconda/argument_parsing.py]: Fixed.

**Comment by poncovka** [comment]
Updated.

**Review by M4rtinK — APPROVED** [review]
Looks good to me. :)

**Pull request merged**